<?xml version="1.0"?>
<launch>
    <param name ="/use_sim_time" value="true"/>
    <!--sim_planner1-->
<!--Input######################################################################################################-->
    <node pkg="rosbag" type="play" name="rosbag" output="log" args="$(find mlmapping)/bag/flight_1.bag">
    </node>

<!--rviz#######################################################################################################-->
   <node type="rviz" name="rviz" pkg="rviz" args="-d $(find mlmapping)/launch/config/mlmapping.rviz" />

<!--visualization##############################################################################################-->
    <!-- <node pkg="mlmapping" type="visualization" name="visualization" output="screen" /> -->

<!--mlmapping##################################################################################################-->
    <arg name="node_start_delay"  default="1.0" />
    <param name="/mlmapping_configfile" type="string" value="$(find mlmapping)/launch/config/config2.yaml"/>

    <!-- Manager -->
    <node pkg="nodelet" type="nodelet"
          name="mlmapping_nodelet_manager" args="manager" output="screen">
          <param name="num_worker_threads" value="2" />
    </node>

    <node pkg="nodelet" type="nodelet" args="load mlmapping/MlMapNodeletClass mlmapping_nodelet_manager"
        name="Mlmap_loader" output="screen"
        launch-prefix="bash -c 'sleep 1; $0 $@' ">
        
        <remap from="/mlmapping/depth"       to="/camera/depth/image_rect_raw"/>
        <remap from="/mlmapping/odom"        to="/vicon_imu_ekf_odom"/>
        <remap from="/mlmapping/imu"        to="/mavros/imu/data"/>
    </node>



</launch>







